package.json

```
@@ -15,10 +15,12 @@
     "test": "stencil test --spec --e2e",
     "test.watch": "stencil test --spec --e2e --watchAll",
     "prepare": "npm run build",
```

Keep the snyk-protect step.

```suggestion
    "prepare": "npm run snyk-protect; npm run build",
```